### PR TITLE
Website Tweaks

### LINKED ISSUE #7: Font change
The font is too hard for your eyes just after you've seen some nice non-Times font over on Github and the wiki.
care to change it?

### LINKED ISSUE #8: Loom 0.4.x is not mentioned anywhere
loom 0.4.x makes everything much easier  
it should at least be given an honorable mention on the page

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Closes #7 
Closes #8 

## Changes
1. this makes use of a better font. Mostly because I don't like Times.
2. this also pulls the latest version of loom, for those who want to use it. 
3. adds a copy to clipboard button so that you can easily copy either of the two code blocks to your clipboard without accidentally dragging your cursor over the entire page
4. adds a fabric logo

## Why merge this PR
1. It doesn't use Times, which is not a sans-serif font 
2. Loom's latest version is also pulled
3. Accidentally dragging your cursor over the entire page will now be pointless
4. fabric logo is important

No original code has been deleted, instead, new code has been added over it.
Still uses the same theme and content
The only major change was the loom table

### CODE REVIEW

**Comment by BoogieMonster1O1** [comment]
https://boogiemonster1o1.github.io/fabric.html
here's a link to the website

**Comment by UpcraftLP** [comment]
you should probably break this PR up into separate pieces, rather than doing one large bulk PR of features

File: fabric.html [hunk 1]
```diff
@@ -5,27 +5,59 @@
     <meta charset="utf-8">
     <title>Fabric Versions</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.14.2/styles/default.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<style>
```
Comment (UpcraftLP) [inline fabric.html:10]: is there any reason to use inline styles here, instead of a separate CSS file?

File: fabric.html [hunk 2]
```diff
@@ -34,45 +66,67 @@ <h3>build.gradle</h3>
...
+    </div>
     <hr>
 
     
-    <h1>Mappings Migration</h1>
+    <h2>Mappings Migration</h2>
     <p>Mappings can be auto updated by using the following command. See the <a href="https://fabricmc.net/wiki/tutorial:migratemappings">wiki page</a> for more help.</p>
     <div name="code">
-        <pre><code class="bash">gradlew migrateMappings --mappings "{yarn_version}"</code></pre>
+        <pre><code class="bash">./gradlew migrateMappings --mappings "{yarn_version}"</code></pre>
```
Comment (UpcraftLP) [inline fabric.html:91]: this change seems unnecessary to me, on windows the leading `./` is not required. Is there a reason to sneak this change in here?

File: fabric.html [hunk 3]
```diff
@@ -34,45 +66,67 @@ <h3>build.gradle</h3>
...
 fabric_version={fabric_version}</code></pre>
     </div>
-
-    <br>
+    </div>
     <hr>
 
     
-    <h1>Mappings Migration</h1>
+    <h2>Mappings Migration</h2>
```
Comment (UpcraftLP) [inline fabric.html:88]: why the change from `h1` to `h2`?
you do this in several places for seemingly no reason at all

File: fabric.html [hunk 4]
```diff
@@ -34,45 +66,67 @@ <h3>build.gradle</h3>
...
     <p>Want to help improve this page? Fork it on <a href="https://github.com/modmuss50/fabric-versions">github</a></p>
+    <p>Note: Copying to clipboard will not work on browsers that support HTML 5 (most browsers do).<br>
+    It will be a problem only if you're using internet explorer, or another old browser.</p>
 
 </body>
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.14.2/highlight.min.js"></script>
 <script charset="UTF-8"
     src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/gradle.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
```
Comment (UpcraftLP) [inline fabric.html:124]: do we really need all of these scripts? what is each of them used for?

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Garbage
```
Comment (UpcraftLP) [inline .gitignore:1]: `garbage` is not a sensible comment.. maybe use something like `macOS-specific files`?

**Comment by UpcraftLP** [comment]
> here's a link to the website

that link does not work and leads to a 404 page.

File: .gitignore (hunk 5 above)
Comment (BoogieMonster1O1) [reply .gitignore:1]: > > here's a link to the website
> 
> that link does not work and leads to a 404 page.

Fixed.

Comment (BoogieMonster1O1) [reply .gitignore]: ~~It kinda is garbage though~~
Fixed.

**Comment by BoogieMonster1O1** [comment]
> > here's a link to the website
> 
> that link does not work and leads to a 404 page.

Fixed

**Comment by modmuss50** [comment]
I dont plan on merging this, as I plan to replace the whole page with something offical on the main fabric website along with the new website.

Sorry about that, there are also one or two other PRs that redesgined the site and it would be unfair on those.

**Comment by BoogieMonster1O1** [comment]
Imma close this PR now
forgot the message those two times
